src/main/java/jenkins/plugins/logstash/LogstashInstallation.java

```diff
@@ -85,6 +90,9 @@ public boolean configure(StaplerRequest req, JSONObject formData) throws FormExc
     }
 
     @Override
+    @SuppressFBWarnings(
+      value="NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE",
+      justification="TODO: investigate")
```

Comment:
can't we just leave those cases unsuppressed? or is the build setup to fail in that case?

Reply:
build fails

Reply:
ok, I think we should disable this on maven level then -- just make it emit warnings for now

let me know if you need help with this

Reply:
project automated CI will will show a failure. Is there a way to make it not fail but just warn?

Reply:
this should be set somewhere and maven and overridable -- let me check

Reply:
we have the FB issues fixed in #43 and related PRs #44 #45 #46 
maybe you can integrate those into your PR instead?

Reply:
Will look into that once I catch up with the holiday pile up of tasks.